src/Shared/PackageManagers/PyPIProjectManager.cs

```diff
@@ -139,12 +146,151 @@ public override async Task<IEnumerable<string>> EnumerateVersions(PackageURL pur
             }
         }
 
+        public override async Task<PackageMetadata> GetPackageMetadata(PackageURL purl)
+        {
+            PackageMetadata metadata = new PackageMetadata();
+            string? content = await GetMetadata(purl);
+            if (string.IsNullOrEmpty(content)) { return metadata; }
+
+            // convert NPM package data to normalized form
+            JsonDocument contentJSON = JsonDocument.Parse(content);
+            JsonElement root = contentJSON.RootElement;
+
+            JsonElement infoElement = root.GetProperty("info");
+
+            metadata.Name = Utilities.GetJSONPropertyStringIfExists(infoElement, "name");
+            metadata.Description = Utilities.GetJSONPropertyStringIfExists(infoElement, "description");
+            string? summary = Utilities.GetJSONPropertyStringIfExists(infoElement, "summary");
+            if (summary?.Length > metadata.Description?.Length)
```

Comment:
This behavior would be confusing in the data if you didn't know the source code.

Reply:
Changed to             if (string.IsNullOrWhiteSpace(metadata.Description))